internal/services/paloalto/next_generation_firewall_vhub_panorama_resource_test.go

```diff
@@ -109,6 +110,29 @@ func (r NextGenerationFirewallVHubPanoramaResource) Exists(ctx context.Context,
 }
 
 func (r NextGenerationFirewallVHubPanoramaResource) basic(data acceptance.TestData) string {
+	if !features.FivePointOhBeta() {
+		return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+%[1]s
+
+resource "azurerm_palo_alto_next_generation_firewall_virtual_hub_panorama" "test" {
+  name                   = "acctest-ngfwvh-%[2]d"
+  resource_group_name    = azurerm_resource_group.test.name
+  location               = azurerm_resource_group.test.location
+  panorama_base64_config = "%[3]s"
+  plan_id                = "panw-cngfw-payg"
+
+  network_profile {
+    virtual_hub_id               = azurerm_virtual_hub.test.id
+    network_virtual_appliance_id = azurerm_palo_alto_virtual_network_appliance.test.id
+    public_ip_address_ids        = [azurerm_public_ip.test.id]
+  }
+}
+`, r.template(data), data.RandomInteger, os.Getenv("ARM_PALO_ALTO_PANORAMA_CONFIG"))
+	}
```

Comment:
Same remark here, optional properties shouldn't be added to basic tests, I think this can be removed since it's been added to the `complete` config. This comment applies to all the other resource tests that have been modified in this PR.

Reply:
Given service team has deprecated the default value "panw-cloud-ngfw-payg", so we have to explicitly set it with new default value. Otherwise it would fail with below error.

Note: All related test cases on Teamcity Daily Run have been failing with below same error for a long time.

```
Error: creating Firewall (Subscription: "xxxxx"
Resource Group Name: "acctestRG-PANGFWVN-250116020412783950"
Firewall Name: "acctest-ngfwvn-250116020412783950"): performing CreateOrUpdate: unexpected status 400 (400 Bad Request) with error: PaymentRequired: SaaS Purchase Payment Check Failed as validationResponse was {"isEligible":false,"errorMessage":"Plan 'panw-cloud-ngfw-payg' is defined as stop sell in offer 'pan_swfw_cloud_ngfw'."}
```